src/helpers/setup.js

```diff
@@ -4,11 +4,21 @@ const {
 	runPostScript
 } = require("../workflows/steps/index.js");
 const getCurrentBranch = require("./getCurrentBranch");
+const getRootDirectory = require("./getRootDirectory");
 const filterFlowBasedOnDevelopBranch = require("./filterFlowBasedOnDevelopBranch");
 const utils = require("../utils");
 
 const setup = async options => {
-	options.configPath = options.config || "./package.json";
+	const rootDir = await getRootDirectory();
+	options.filePaths = {
+		rootPath: rootDir,
+		configPath: `${rootDir}/package.json`,
```

Comment:
Does this accidentally remove support for the `--config` option providing a different path?

Reply:
It does, I thought we had actually remove the ability to set that before but I think it's just that no one used it. Ha. I'll update it to allow for it as it's still a accepted thing!

Reply:
the `--config` option was a PR that originated from Nathan https://github.com/LeanKit-Labs/tag-release/pull/39 who wanted `tag-release` to support `manifest.json` instead of `package.json` so he could publish a Chrome Extension.